app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,15 @@
 <resources>
     <string name="app_name">ColorNoteTaking</string>
+    <string name="search_title">Search your note</string>
```

Comment:
tên resource e nên đặt cho clear và dễ hiểu hơn
tring names start with a prefix that identifies the section they belong to. We use <HOW>_<DESCRIPTION> for strings naming. <HOW> to indicate reason of the string will be used & description give any extra information.

ví dụ
<-- Search Screen ->
```suggestion
    <string name="search_hint_title">Search your note</string>
```